client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -332,21 +326,29 @@ class ListItem extends PureComponent {
 
   renderDescription = () => {
     const {
-      data: { _id: itemId, description, isOrdered, name },
+      data: { _id: itemId, description, isOrdered, name, busy },
       isMember
     } = this.props;
     const isFieldDisabled = !isMember || isOrdered;
+    let isDescriptionBusy;
+
+    if (busy) {
+      const { descriptionBusy } = busy;
+
+      isDescriptionBusy = descriptionBusy;
```

Comment:
Can we use descriptionBusy directly instead of creating new alias?

Reply:
how can I get access to `descriptionBusy` variable outside if block without aliasing?